catalog.html

```diff
@@ -0,0 +1,404 @@
+          <p>Закладки:</p>
+          <output class="bookmarks-value">0</output>
+        </a>
+        <a class="header-basket" href="shop-basket.html">
+          <img src="img/nav-site-icons/header/icon-cart.svg" width="15px" height="15px" alt="иконка-корзина">
+          <p>Корзина:</p>
+          <output class="basket-value">0</output>
+        </a>
+        <a class="header-checkout" href="checkout.html">Оформить заказ</a>
+      </div>
+    </div>
+    <div class="2nd-floor-header">
+      <h1>Интернет-магазин строительных материалов и инструментов для ремонта</h1>
+      <div class="header-contacts">
+        <p class="contacts header-tel-number">
+          <img src="img/nav-site-icons/header/icon-phone.svg" width="19px" height="19px" alt="телефон">
+          +7(812)555-05-55
+        </p>
+        <p class="contacts header-city-adress">Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p>
+      </div>
+      <div class="uac-header">
+        <section class="user-account-control">
+          <a class="user-profile" href="user-profile.html">
+            <img src="img/nav-site-icons/header/icon-user.svg" width="20px" height="18px" alt="иконка-профиля">
+            <span class="user-profile-name">Равшан Джамшутович</span>
+          </a>    
+          <a class="user-logout" href=#>
+                <img src="img/nav-site-icons/header/icon-logout.svg" width="20px" height="17px">
+          </a>
+          <p class="user-mainmenu">
+            <a class="my-orders" href="my-orders.html">
+              Мои заказы
+            </a>
+            <a class="personal-area" href="user-profile.html">
+              Личный кабинет
+            </a>
+          </p>
+        </section>
+      </div>
+    </div>
+    <div class="3rd-floor-header">
+      <nav class="main-navigation">
+        <ul class="site-navigation">
+          <li><a href="index.html">Главная</a></li>
+          <li><a href="about-conpany.html">Компания</a></li>
+          <li><a href="catalog.html">Каталог</a></li>
+          <li><a href="news.html">Новости</a></li>
+          <li><a href="special-offers.html">Спецпредложения</a></li>
+          <li><a href="delivery.html">Доставка</a></li>
+          <li><a href="Contacts.html">Контакты</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+    
+  <main class=container>
+    <div class="1st-floor-main-catalog">
+      <div class="breadcrumps-catalog">
+        <ul class="breadcrumbs">
+          <li><a href="index.html"><img src="img/nav-site-icons/icon-home.svg" alt="Домой"></a></li>
+          <li><a href="catalog.html">КАТАЛОГ</a></li>
+          <li><a href="tool.html">ИНСТРУМЕНТ</a></li>
+          <li class="breadcrumbs-current">ПЕРФОРАТОРЫ</li>
+        </ul>
+      </div>
+    </div>
+    <div class="2nd-floor-main-catalog">
+      <h2>ПЕРФОРАТОРЫ</h2>
+    </div>
+    <div class="3rd-floor-main-catalog">
+      <div class="filter">
+        <div class="filter-head">
```

Comment:
а зачем для заголовка обертка из дива?

Reply:
```suggestion
        <div class="filter-head">
```
это действительно непонятно, потому что он относится к блоку с ценой и прочим, в общем убрал это недоразумение)